### PR TITLE
Move predicate_label examples to predicate_id in sssom_schema.yaml

### LINKED ISSUE #369: Docs: `predicate_label` examples -> `predicate_id`
The examples section in the `predicate_label` [page](https://mapping-commons.github.io/sssom/predicate_label/) looks like it should be in the `predicate_id` [page](https://mapping-commons.github.io/sssom/predicate_id/).

### PR DESCRIPTION
**Pull request by matentzn**
Fixes #369.

The examples for the predicate_id slot where in the wrong slot (predicate_label), here we move them to be correct.

I have also added a single example for the predicate_label slot.

- [X] `docs/` have been added/updated if necessary
- [ ] `make test` has been run locally
- [ ] tests have been added/updated (if applicable)
- [ ] [CHANGELOG.md](https://github.com/mapping-commons/sssom/blob/master/CHANGELOG.md) has been updated.

If you are proposing a change to the SSSOM metadata model, you must 

- [ ] provide a full, working and valid example in `examples/`
- [ ] provide a link to the related GitHub issue in the `see_also` field of the linkml model
- [ ] provide a link to a valid example in the `see_also` field of the linkml model

### CODE REVIEW

File: src/sssom_schema/schema/sssom_schema.yaml [hunk 1]
```diff
@@ -221,6 +209,20 @@ slots:
         description: Two terms are related in some way. The meaning is frequently consistent across a single set of mappings. Note this property is often overloaded even where the terms are of a different nature (e.g. interpro2go)
       - value: rdfs:seeAlso
         description: The subject and the object are associated in some unspecified way. The object IRI often resolves to a resource on the web that provides additional information.
+  predicate_modifier:
+    description: A modifier for negating the prediate. See https://github.com/mapping-commons/sssom/issues/40 for discussion
```
Comment (joeflack4) [inline src/sssom_schema/schema/sssom_schema.yaml:213]:
```suggestion
    description: A modifier for negating the predicate. See https://github.com/mapping-commons/sssom/issues/40 for discussion
```

File: src/sssom_schema/schema/sssom_schema.yaml [hunk 2]
```diff
@@ -183,19 +183,7 @@ slots:
...
-    see_also:
-      - https://github.com/mapping-commons/sssom/issues/107
-    examples:
-      - value: Not
-        description: Negates the predicate, see documentation of predicate_modifier_enum
-  predicate_label:
-    description: The label of the predicate/relation of the mapping
-    range: string
+    
     examples:
```
Comment (joeflack4) [inline src/sssom_schema/schema/sssom_schema.yaml:187]: LGTM!

**Review by joeflack4 — APPROVED** [review]

**Review by ehartley — APPROVED** [review]
Looks good to me.

**Review by joeflack4 — APPROVED** [review]

**Review by hrshdhgd — CHANGES_REQUESTED** [review]
run `make all` ... same as https://github.com/mapping-commons/sssom/pull/386#pullrequestreview-2217867625

**Pull request merged**